Add a section how to add 3d model in the asset store. (#100)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/community/contribute-to-the-assets-store.md

`TAGS.md` files will work like for images. Sub folders are also used for tags as described for images.

## 3D models

Supported 3D file formats are:

- .glb
- .blend
- .gltf
- .obj
- .fbx
- .x3d
- .vrml

**Animations** are taken into account and will be displayed in the store.
Be sure to use logical animation names such as `run`, `jump`, `idle`, etc.

**Dimensions** of 3d models must be consistent across a pack. For instance, props should fit characters hands.

**The point of origin** is important, it will be used as the position and rotation center of object in GDevelop.

> **Textures** are supported only in format types that are contained, such as `.glb`, `.blend`, `.fbx`.

Reviewer: I'm not sure it matters as Blender should be able to find the images and include them when it exports to GLB, but none of the free packs had images.

Author: This demanding for manual tests.

Author: I tested it, and the texture are converted and include during the convertion.

Reviewer: Should this line be removed then?

!!! note
    

>     There's no need to add images preview for the models, the asset store will automatically generate them.

Reviewer: But a thumbnail for the pack is needed.

Author: Indeed, that will be mentioned in another section of this page. This way I could add a link from my best practice guide to the wiki. I'll take a look this weekend. We can for now merge this PR about 3D models.

`TAGS.md` files will work like for images. Sub folders are also used for tags as described for images.

## 3D models

Supported 3D file formats are:

- .glb
- .blend
- .gltf
- .obj
- .fbx
- .x3d
- .vrml

**Animations** are taken into account and will be displayed in the store.

> Be sure to use logical animation names such as `run`, `jump`, `idle`, etc.

Reviewer: The full list is in `AnimationsOrder.js`.
Do you think we should give the full list?
The list can probably be refined if you want.

Author: There's no need to bore people with it, the most important thing is that the name is logic and short that is the purpose of the few examples.

In the content moderation/verification process for a future automatic form, it's an error that could be returned to say hey this animation name is weird please fix it.

Reviewer: If creators have already made the models, we shouldn't ask them to change them. Long names are not a issue as there is a mapping. What is important is that most models have the same animation names when it fits.
For instance, "Armature|Idle" is not an issue as long as it's consistent.

> **Scale/dimensions** of 3d models, be consistent, a building is always bigger than a character.

Reviewer: Our buildings are actually on a different scale than characters because they probably have different usages: platformers vs simulation. It's not perfect, but it's better than having 10 pixels characters or building of 1000 pixels.

```suggestion
**Dimensions** of 3d models must be consistent across a pack. For instance, props should fit characters hands.
```

> **The point of origin** is important, it will be used as a rotation point by GDevelop.

Reviewer: ```suggestion
**The point of origin** is important, it will be used as the object position and center of rotation in GDevelop.
```

**Textures** are supported only in format types that are contained, such as `.glb`, `.blend`, `.fbx`.

> **Texture** formats must be `.png` or `.jpg`.

Reviewer: Isn't this a contradiction?

Author: I don't know.
What happens when in a binary .fbx there is a .TIFF image, is the 3D model and texture are converted correctly into .GLB and is the texture is visible on the model once imported into GDevelop?

If so, I'll have to delete this line, which is of no use.

Reviewer: I guess Blender convert what need to be converted and Three.js handle the full spec of GLB.

Author: Alright the import/export script from Blender is taking care of that, this is nice.

> **Texture** formats must be `.png` or `.jpg`.

Author: ```suggestion
```

## For fonts

Put the fonts in to a `Font` folders.
Formats supported are:

- .ttf
- .otf

## 3D models

Supported 3D file formats are:

- .glb
- .blend
- .gltf
- .obj
- .fbx
- .x3d
- .vrml

> 
> **Animations** are taken into account and will be displayed in the store.

Reviewer: Should it be a list?
```suggestion
Best practices:
- **Animations** are taken into account and will be displayed in the store.
```

Reviewer: Why not?

> **Animations** are taken into account and will be displayed in the store.
> Be sure to use logical animation names such as `run`, `jump`, `idle`, etc.

Reviewer: ```suggestion
**Animation names** should be meaningful such as `run`, `jump`, `idle`, etc.
```
